cmd/semaphore/generate/protobuf/protobuf.go

```diff
@@ -18,22 +22,27 @@ import (
 	"github.com/spf13/cobra"
 )
 
-var params = config.New()
+var (
+	params = config.New()
+
+	// Command represents the semaphore daemon command
+	Command = &cobra.Command{
+		Use:   "protobuf",
+		Short: "Generates a protobuf definitions",
+		Long: `Generates a protobuf definitions.
+	*NOTE*: This is a experimental feature and not all features are supported yet.`,
+		RunE:         run,
+		SilenceUsage: true,
+	}
 
-// Command represents the semaphore daemon command
-var Command = &cobra.Command{
-	Use:   "protobuf",
-	Short: "Generates a protobuf definitions",
-	Long: `Generates a protobuf definitions.
-*NOTE*: This is a experimental feature and not all features are supported yet.`,
-	RunE:         run,
-	SilenceUsage: true,
-}
+	output string
+)
 
 func init() {
 	Command.PersistentFlags().StringSliceVar(&params.Protobuffers, "proto", []string{}, "If set are all proto definitions found inside the given path passed as schema definitions, all proto definitions are also passed as imports")
 	Command.PersistentFlags().StringSliceVarP(&params.Files, "file", "f", []string{"config.hcl"}, "Parses the given file as a definition file")
 	Command.PersistentFlags().StringVar(&params.LogLevel, "level", "warn", "Global logging level, this value will override the defined log level inside the file definitions")
+	Command.PersistentFlags().StringVarP(&output, "output", "o", "", "Output directory")
```

Comment:
Maybe it is a good idea to add a small warning that all missing directories are created?